packages/nimble-components/src/select/index.ts

```diff
@@ -507,22 +510,30 @@ export class Select
      */
     public inputHandler(e: InputEvent): boolean {
         this.filter = this.filterInput?.value ?? '';
-        this.filterOptions();
+        if (this.filterMode === FilterMode.standard) {
+            this.filterOptions();
 
-        const enabledOptions = this.filteredOptions.filter(o => !o.disabled);
-        let activeOptionIndex = this.filter !== ''
-            ? this.openActiveIndex ?? this.selectedIndex
-            : this.selectedIndex;
+            const enabledOptions = this.filteredOptions.filter(
+                o => !o.disabled
+            );
+            let activeOptionIndex = this.filter !== ''
+                ? this.openActiveIndex ?? this.selectedIndex
+                : this.selectedIndex;
 
-        if (
-            enabledOptions.length > 0
-            && !enabledOptions.find(o => o === this.options[activeOptionIndex])
-        ) {
-            activeOptionIndex = this.options.indexOf(enabledOptions[0]!);
-        } else if (enabledOptions.length === 0) {
-            activeOptionIndex = -1;
+            if (
+                enabledOptions.length > 0
+                && !enabledOptions.find(o => o === this.options[activeOptionIndex])
+            ) {
+                activeOptionIndex = this.options.indexOf(enabledOptions[0]!);
+            } else if (enabledOptions.length === 0) {
+                activeOptionIndex = -1;
+            }
+            this.setActiveOption(activeOptionIndex);
+        }
+
+        if (this.filterMode !== FilterMode.none) {
+            this.emitFilterInputEvent();
```

Comment:
In [the storybook build](https://60e89457a987cf003efc0a5b-cllvgqkkug.chromatic.com/?path=/story/components-select--select&args=filterMode:standard) in the Actions tab, I'm seeing the event be emitted twice for each key press. I know Storybook used to have bugs that would cause this but I haven't seen them recently. Can you check if that's a real issue or a Storybook issue? 

I'm also seeing `change` be fired twice so I suspect Storybook, but worth confirming.

Reply:
Fixed.